h2o-r/h2o-package/R/connection.R

```diff
@@ -148,6 +149,10 @@ h2o.init <- function(ip = "localhost", port = 54321, name = NA_character_, start
     stop("`cacert` must be a character string or NA_character_")
   if(!is.logical(insecure) || length(insecure) != 1L || is.na(insecure))
     stop("`insecure` must be TRUE or FALSE")
+  if(!is.logical(use_spnego) || length(use_spnego) != 1L || is.na(use_spnego))
```

Comment:
We could generalize these check in one place somewhere in future. Just saying.

Reply:
@honzasterba this option is a good candidate to have in ~./h2oconfig - please add reading the value from the config to the h2o.init function (see eg. `init.cacert`)